Client/src/Pages/Incidents/index.jsx

```diff
@@ -1,9 +1,38 @@
-import React from 'react'
+import React from "react";
+import "./index.css";
+import { Button } from "@mui/material";
+import CheckCircleOutlineIcon from "@mui/icons-material/CheckCircleOutline";
 
 const Incidents = () => {
-    return (
-        <div>Incidents</div>
-    )
-}
+  const maintenanceItems = [
+    { id: 1, text: "Mark your maintenance periods" },
+    { id: 2, text: "Eliminate any misunderstandings" },
+    { id: 3, text: "Stop sending alerts in maintenance windows" },
+  ];
 
-export default Incidents
\ No newline at end of file
+  return (
+    <div className="maintenance-checklist-main">
+      <div className="maintenance-image"></div>
```

Comment:
On Figma there is an option to export/ copy frames or components as svgs, so you can actually take the skeleton frame from the design and use it as the background for `maintenance-image`.

Reply:
Thanks for your comment.